### DISCUSSION TITLE
Feature request for `muted-tests.json`

### DISCUSSION THREAD

**Discussion by vovsemenv** [allure-framework/allure2#2351] [TestOps]
When make tests as muted in the `TestOps` interface to mark them as muted, but they will be still failed in the CI/CD.
If allure give the way to fetch muted tests in the following format just like `testplan.json` we can mark tests as `should fail` in the integrations without making changes to tests.

Feature request from users:
- https://t.me/playwright/41292

**Comment by cheshi-mantu** [comment]
@vovsemenv hello sir,
the right place for FR is https://help.qameta.io
However, your task can be fulfilled using existing functionality of the selective test run.
If you mute test cases on Allure Testops side, then you can filter out such test cases using [AQL](https://docs.qameta.io/allure-testops/advanced/aql/) and create a test plan based on such filter:

```shell
muted = false
``` 

and when executing this test plan from Allure Testops you will get on test framework side the list of tests that are not muted in Allure Testops

![image](https://github.com/allure-framework/allure2/assets/8607348/1a9bb127-f6e5-43a5-b75c-38ead8ad7207)

**Reply by cheshi-mantu** [reply]
however, this approach will leave you blind, i.e. you won't see if the muted test was fixed as you won't run it

**Reply by cheshi-mantu** [reply]
using the muted attribute allows you triggering non-muted tests via Allure Testops UI as well as triggering these via allurectl https://docs.qameta.io/allure-testops/ecosystem/allurectl/#creating-a-launch-based-on-test-plan-created-in-allure-testops-project
e.g.

```shell
export ALLURE_TESTPLAN_PATH="testplan.json"
....

./allurectl test-case plan -q "muted = false" --output-file ${ALLURE_TESTPLAN_PATH}
./allurectl watch -- runtests_here

```

this will create `testplan.json` with the tests that are not muted in Allure Testops
